.gitignore

```diff
@@ -31,8 +31,8 @@ yarn-error.log*
 .env.development.local
 .env.test.local
 .env.production.local
+.env.sample
+.env.local.sample
```

Comment:
Shouldn't we commit the sample files so people can copy them and know what to set up?

Reply:
@KalisaFalzone sorry this is a duplicate PR, I meant to close it. Yes the sample env files have been committed on the staging branch. But we want to prevent contributors from accidentally pushing further changes to the sample env files.

Closing this PR now.